doc/visual-programming/source/widgets/data/pivot.md

```diff
@@ -0,0 +1,68 @@
+Pivot Table
+===========
+
+Reshape data table based on column values.
+
+**Inputs**
+
+- Data: input data set
+
+**Outputs**
+
+- Pivot Table: contingency matrix as set in the widget
+- Filtered Data: subset selected from the plot
+- Grouped Data: data table grouped by row values
+
+**Pivot Table** summarizes the data of a more extensive table into a table of statistics. The statistics can include sums, averages, counts, etc. The widget also allows selecting a subset from the plot and grouping by row values, which have to be a discrete variable. Data with only numeric variables cannot be displayed in the plot.
+
+![](images/Pivot-stamped.png)
+
+1. Discrete or numeric variable that will be used for row values. Numeric variables are considered as integers in this case. Variable values will appear as rows in the table.
+2. Discrete variable that will be used for column values. Variable values will appear as columns in the table.
+3. Values that will be used for aggregation. Aggregated values will appear as cells in the table.
+4. Aggregation methods:
+   - For any variable type:
+      - *Count*: number of instances that appear in the data
```

Comment:
`- *Count*: size of the group, that is, the number of instances with the given row and column value.`

I'm not totally sure this is better, though. :)